en/config_rover/ackermann.md

```diff
@@ -30,70 +28,201 @@ To configure the Ackermann rover frame and outputs:
-| <a id="RA_WHEEL_BASE"></a>[RA_WHEEL_BASE](../advanced_config/parameter_reference.md#RA_WHEEL_BASE)       | Wheel-base of the rover which is measured from the back to the front wheel | m    |
-| <a id="RA_MAX_STR_ANG"></a>[RA_MAX_STR_ANG](../advanced_config/parameter_reference.md#RA_MAX_STR_ANG)    | Maximum steering angle of the rover                                        | deg  |
-| <a id="RA_MISS_VEL_DEF"></a>[RA_MISS_VEL_DEF](../advanced_config/parameter_reference.md#RA_MISS_VEL_DEF) | Default velocity the rover will drive during the mission                   | m/s  |
+However, this mode is also affected by the steering slew rate and acceleration/deceleration limits. This configuration becomes mandatory for subsequent modes, which is why we do this setup here.  
+Navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
 
-![Geometric parameters](../../assets/airframes/rover/rover_ackermann/geometric_parameters.png)
+1. [RA_WHEEL_BASE](#RA_WHEEL_BASE) [m]: Measure the distance from the back to the front wheels.
+2. [RA_MAX_STR_ANG](#RA_MAX_STR_ANG) [deg]: Measure the maximum steering angle.
 
-This is enough to start driving missions, but depending on the rover might not yet lead to satisfactory performance .
-If that is the case further tuning is required which is outlined in [Mission parameters](#mission-parameters).
+   ![Geometric parameters](../../assets/airframes/rover/rover_ackermann/geometric_parameters.png)
 
-## Tuning (Advanced)
+3. [RA_MAX_THR_SPEED](#RA_MAX_THR_SPEED) [m/s]: Drive the rover at full throttle and set this parameter to the observed value of the ground speed.
 
-To get an overview of all parameters that are related to the Ackermann rover module navigate to the _Rover Ackermann_ group in the _Parameters_ section of QGroundControl.
+   :::info
+   This parameter is also used for the feed-forward term of the speed control. It will be further tuned in the configuration of [Position mode](#position-mode).
+   :::
 
-### General Parameters
+4. [RA_MAX_ACCEL](#RA_MAX_ACCEL) [m/s^2]: Maximum acceleration you want to allow for your rover.
 
-These parameters affect the general behaviour of the rover.
-This will influence both auto and manual modes.
+   <a id="RA_MAX_ACCEL_CONSIDERATIONS"></a>
 
-| Parameter                                                                                       | Description                                | Unit |
-| ----------------------------------------------------------------------------------------------- | ------------------------------------------ | ---- |
-| <a id="RA_MAX_SPEED"></a>[RA_MAX_SPEED](../advanced_config/parameter_reference.md#RA_MAX_SPEED) | Speed the rover drives at maximum throttle | m/s  |
+   :::tip
+   This value is set to your preference, there is no general rule of thumb since it depends entirely on your rover and use case.
```

Comment:
In some senses your tips that follow _are_ equivalent to a rule of thumb. I'd simply say "The value depends on your rover and use case". I'm subediting now, but if I miss some, feel free to edit :-)

Reply:
Agreed, found one more instances of this that I adjusted here: [32ad899](https://github.com/PX4/PX4-user_guide/pull/3407/commits/32ad89974d3157051edd5a79d345ff84fd5226f4)